nncf/torch/quantization/init_range.py

```diff
@@ -115,10 +116,20 @@ def convert_reduction_shape(self, per_sample_stats) -> ReductionShape:
         if self._per_channel:
             val = (ndims + self._channel_idx) % ndims
             reduction_shape.remove(val)
+            if not val and self.use_per_sample_stats(per_sample_stats):
+                raise RuntimeError("Batch dimension should be equal to zero")
         if self.use_per_sample_stats(per_sample_stats):
             reduction_shape = reduction_shape[1:]  # Assumes batch is the first dimension
         return tuple(reduction_shape)
 
+    def convert_statistic_params(self, per_sample_stats):
+        reducer_axes = self.convert_reduction_axes(per_sample_stats)
+        aggregator_axes = (0, 1) if self.use_per_sample_stats(per_sample_stats) else (0,)
+        return {
+            "reducers_axes": reducer_axes,
+            "aggregators_axes": aggregator_axes,
+        }
```

Comment:
Why `reducers_axes` while you've calculated `reduction_axes`?

Reply:
Done